tests/e2e/tests/pilot/externalservice_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pilot
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestExternalService(t *testing.T) {
+	if !tc.V1alpha3 {
+		t.Skipf("Skipping %s: v1alpha3=false", t.Name())
+	}
+
+	// This list is ordered so that cases that use the same egress rule are adjacent. This is
```

Comment:
There is another skip in the egress rules ( since it's flaky and slow - it shouldn't run in circle presubmit, only post submit.

Reply:
Is it flaky just due to relying on external resources (httpbin, wikipedia, etc)? If that's the case, I should add it to this test.

Reply:
Is there a standard go testing way to exclude tests that we could leverage rather than adding more special args?

Reply:
there is an Egress flag